### ISSUE TITLE
Does scrapy.Requests has streaming method?

### ISSUE THREAD

**Issue by lycanthropes** [scrapy/scrapy#3880]
I want download many PDF files, and I want to use scrapy to complete this task. But I find scrapy couldn't download them completely, there are always 1KB files in the result. So I think it is necessary to use a method such as requests.streaming in requests module.

**Comment by kmike** [comment]
Yes, that'd be a great feature to add.

**Comment by lycanthropes** [comment]
That is good. Thank you for listening to my advice ,and I am looking forward to it.

**Comment by Gallaecio** [comment]
A potential use case would be to only fetch the response body until the desired data is found, reducing network traffic: https://stackoverflow.com/q/59541755/939364

**Comment by chetan1029** [comment]
Yes most of the time we just need to scrap the title of the page so if is it possible to stream the data in chunks and skip the rest of the chunks.
